deployments/helm/nvidia-dra-driver-gpu/values.yaml

```diff
@@ -88,6 +88,8 @@ kubeletPlugin:
   podSecurityContext: {}
   nodeSelector: {}
   tolerations: []
+  kubeletRegistrarDirectoryPath: /var/lib/kubelet/plugins_registry
+  kubeletPluginsDirectoryPath: /var/lib/kubelet/plugins
```

Comment:
Just out of curiosity, was this in the original PR from the example driver? I'm not sure is not where I would have added this, but we should stay consistent if this is where it is there.